src/lib/Components/__tests__/FilterModal-tests.tsx

```diff
@@ -1,22 +1,82 @@
-import { Theme } from "@artsy/palette"
-import { FilterModal } from "lib/Components/FilterModal"
+import { Sans, Theme } from "@artsy/palette"
+import { SortOptionsScreen } from "lib/Components/ArtworkFilterOptions/SortOptions"
+import { FakeNavigator as MockNavigator } from "lib/Components/Bidding/__tests__/Helpers/FakeNavigator"
 import React from "react"
-import "react-native"
-import * as renderer from "react-test-renderer"
+import { act, create } from "react-test-renderer"
+import { FilterHeader, FilterOptions } from "../FilterModal"
 
-it("renders a snapshot of the filter artworks modal", () => {
-  let props
+let mockNavigator: MockNavigator
 
-  props = {
-    visible: true,
-  }
+beforeEach(() => {
+  mockNavigator = new MockNavigator()
+})
+
+describe("Filter options screen", () => {
+  it("renders the Filter Options screen properly", () => {
+    const FilterOptionsScreen = create(
+      <Theme>
+        <FilterOptions navigator={mockNavigator as any} closeModal={jest.fn()} />
+      </Theme>
+    )
+    expect(FilterOptionsScreen).toMatchSnapshot()
+  })
+})
+
+describe("Filter modal navigation flow", () => {
+  it("allows users to navigate forward to sort screen from filter screen", () => {
+    let filterScreen = create(
+      <Theme>
+        <FilterOptions closeModal={jest.fn()} navigator={mockNavigator as any} />
+      </Theme>
+    )
+    const filterListItem = filterScreen.root.findByProps({ "data-test-filter-item": "item" })
```

Comment:
It seems a little odd to pass props in order to find the item. Can we select for this based on the component's text or class name? (this applies to the other places as well!)